template/Makefile

```diff
@@ -7,11 +7,24 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 
 #### Development ####
 # start commands
-up: clean update-box
+dev: update-box
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--skip-tags "test"' vagrant up --provision
+
+up: update-box
+ifdef CI # CI is set in Github Actions
```

Comment:
I might be missing something, but where in github actions is CI set?

Reply:
https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
Ah, sweet. Thanks